### DISCUSSION TITLE
Copy and paste frame sequences

### DISCUSSION THREAD

**Discussion by PlumpyPotato** [keiwando/toonsquid#272] [Q&A] [answered]
Hello! I can't figure out how to copy and paste entire sequences of frames in the same timeline(and others too), to give an extended looping feel. Does anyone knows if it's actually possible to do it, and if yes, how? I'm in love with this app, and this is the only thing giving me headaches +_+
Images below detailing better what I mean, using Clip Studio Paint's timeline as an example:

![Illustration3](https://user-images.githubusercontent.com/131207750/232937977-34a9156c-e469-4846-b491-a8d3a369ff1e.png)

Thank you for your time~

**Comment by DWrightAnimation** [comment, marked as answer]
What you want to do is double tap the frame quickly to make a blue selection box appear, that way you can select multiple frames at once. Then, tap on one of the frames, select copy, then go to any frame on the timeline, tap and hit paste, presto.

**Reply by PlumpyPotato** [reply]
Thank you so much! I was tapping twice, and it ended up copying and pasting only one frame, that's why it wasn't working, I just needed to tap once... (it was so simple in the end ORZ). Again, thank you, my problem is now solved! :]
